tests/example_scripts/global_terminator_satzen.sh

```diff
@@ -0,0 +1,145 @@
+# # # This source code is subject to the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+#!/bin/bash
+
+###############################
+## Infrared-Day-Only and Infrared-Night-Only Products
```

Comment:
This is a convenience script for producing a bunch of global annotated imagery for a given dataset (to help with tiny sectors creation,  and to just get a quick look at the day/ night/ location composition of a dataset). I commented it in the incoming patch

I also removed support for calling it with no argumentsto get sample output from all satellites.  Now you have to explicitly call with each dataset